src/services/datastore.js

```diff
@@ -29,7 +29,9 @@ export async function queryDatastoreConfig() {
 export async function queryMonthIndices(params) {
   const { datastoreConfig } = params;
 
-  const endpoint = `${datastoreConfig.elasticsearch}/_aliases`;
+  const indices = `/_aliases`;
 
-  return request.get(endpoint);
+  const endpoint = `${datastoreConfig.pbench_server}/elasticsearch`;
```

Comment:
Is there a specific entry in `datastoreConfig` for `pbench_server`? If not, this field will always be undefined. This URL gets you the indices without which the search header won't finish loading, if it won't finish loading, then none of the tests in the Search page will work. That explains why your tests are failing. You might wanna take a look at this endpoint here